docs/schedule-exceptions.adoc

```diff
@@ -0,0 +1,82 @@
+= Schedule exceptions
+
+Schedule exceptions describes any deviation from “regular schedule” generated from KOS data, i.e. state holiday, dean’s holiday, temporary change of room etc.
```

Comment:
It's plural (your favorite `s` in third person); `i.e.` should be `e.g.`; _deviation_ is kind of weighted word, I think _variance_ or _difference_ would be better here.

> Schedule exceptions **describe** **differences** from **the** “regular schedule” generated from KOS data, **e.g.** state holiday, dean’s holiday, temporary change of room etc.